packages/odyssey-react/src/components/Link/Link.module.scss

```diff
@@ -9,6 +9,7 @@
  *
  * See the License for the specific language governing permissions and limitations under the License.
  */
+@value infobox from '../Infobox/Infobox.module.scss';
```

Comment:
The output classes here use encapsulated selectors, right? If so, I think this is a great solve. Link styles live within Link, the styling happens automagically, and we avoid variant fatigue. (The docs are so incomplete, I didn't realize this was an option until combing through stacks of CSS Modules articles.)

Going forward, I think we'll want to keep an eye on these context lists. If we hit three or more planned use cases (aka parent selectors), we'll probably want to talk to @joejezowski-okta about making an official variant for broad use.

Reply:
They do!

Reply:
This is interesting! Thanks for sharing this @edburyenegren-okta @arnoldsandoval-okta. Put this kind of stuff in our main chat to share more often!

I wish that the compile for our build would fail if the exported class wasn't available. It doesn't work the way `composes` does which gives us an extra level of safety we don't get here. It would be nice to know that if the selector from `infobox` changes we would catch that immediately instead of breaking our layout and needing a VRT test for example.

This is an interesting feature because if used incorrectly we can still write lots and lots of contextual selectors that will make our CSS harder to maintain, even if they are using encapsulated class names! For example, if we had instead used `@value link from '../Link/Link.module.scss` within the `Infobox` styles (similar to what we had originally in this PR) we'd be introducing contextual knowledge of the color and font decoration overrides that are necessary for Link which we don't want.

I think we also have a minor `stylelintplus scss/at-rule-no-unknown` violation here to fix before merge.